tutorials/modules/ROOT/pages/failover-clients-with-hazelcast-cloud.adoc

```diff
@@ -0,0 +1,177 @@
+= Configuring a Failover Client
+:url-code-sample-client-failover: https://github.com/hazelcast/hazelcast-cloud-code-samples/tree/master/client-failover
+:description: In this tutorial, you'll update the code in a Java client to automatically connect to a secondary, failover cluster if it cannot connect its original, primary cluster. You'll also run a simple test to make sure that your configuration is correct and then adjust it to include exception handling.
+:page-layout: tutorial
+:page-product: cloud
+:page-categories: Manage Clusters
+:page-est-time: 30 mins
+
```

Comment:
```suggestion
:page-lang: java

```

Allows users to find this tutorial when they filter for those that use Java

Reply:
Updated as requested.